05_iigo/index.tex

```diff
@@ -297,43 +297,44 @@ \section{Constitutional Rights and Obligations in the Archipelago}
 
-The result of monitoring is intended to be a trigger for the initiation of power transfer, whereby a negative result indicates that at least one rule was broken by the role monitored and the role should be re-elected. Each turn, in addition to monitoring the actions taken by the role in that IIGO session, the election that role held in the previous IIGO session, if one was held at all, is checked for compliance with the rules of power-transfer.
+The result of monitoring is intended to be a trigger for the initiation of power transfer, whereby a declaration of a negative result indicates that at least one rule was broken by the role monitored and the role should be re-elected. Each turn, in addition to monitoring the actions taken by the role in that IIGO session, the election that role held in the previous IIGO session, if one was held at all, is checked for compliance with the rules of power-transfer.
 
-Within the scope of the coursework, we decided to pursue only \emph{one degree of monitoring}, meaning that the powers relating to the accountability cycle will not be monitored themselves. We assume that agents will act in the interest of all the islands in the archipelago. Hence, while the agents might be inclined to break the rules to benefit in some form, it is assumed that the others will negatively see any non-compliant islands based on the assumption that the proposed IIGO system is in place to maintain the welfare of all the islands.
+Figure~ \ref{fig:cycles_in_IIGO} illustrates the reverse nature of the monitoring and power transfer cycles. This is a choice made by design to spread out the process of pre-mature removal from power as a result of wrongdoing. The goal is to avoid malice. If it is one role's power to monitor, it is the other's to initiate and facilitate power transfer. Hence the latter is given a second opinion for cases where the declaration of wrongdoing is not truthful.
```

Comment:
```suggestion
Figure~ \ref{fig:cycles_in_IIGO} illustrates the reverse nature of the monitoring and transfer-of-power cycles. This is a design choice specifically made in order to avoid any premature removal of an island from the power as a result of wrongdoing and to avoid malice. If it is one role's power to monitor, it is the other role's power to initiate and facilitate power transfer. Hence the latter is given a second opinion for cases where the declaration of wrongdoing is not truthful.
```

Reply:
```suggestion
Figure~ \ref{fig:cycles_in_IIGO} illustrates the reverse nature of the monitoring and transfer-of-power cycles. This is a design choice made to diversify the islands that hold the powers that enable the process of a role's pre-mature removal from power as a result of wrongdoing, hence helping to avoid malice. If it is one role's power to monitor, it is the other role's power to initiate and facilitate power transfer. Hence the latter is given a second opinion for cases where the declaration of wrongdoing is not truthful.
```

Reply:
ok